distributed/shuffle/arrow.py

```diff
@@ -1,12 +1,12 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, BinaryIO
+from typing import TYPE_CHECKING, Any, BinaryIO
 
 if TYPE_CHECKING:
     import pyarrow as pa
 
 
-def dump_batch(batch, file: BinaryIO, schema=None) -> None:  # type: ignore[no-untyped-def]
+def dump_batch(batch: Any, file: BinaryIO, schema: pa.Schema) -> None:
```

Comment:
I think the "tight" type annotation here is
```
if TYPE_CHECKING:
    from _typeshed import ReadableBuffer
...

def dump_batch(batch: ReadableBuffer, ...)
```

Reply:
That does not work for me. mypy says, it wants just `bytes`. Good enough for me either way

Reply:
Ah :(.